tool/sdg/src/main/java/org/openscience/cdk/layout/AtomPlacer.java

```diff
@@ -417,6 +419,20 @@ public void placeLinearChain(IAtomContainer atomContainer, Vector2d initialBondV
         }
     }
 
+    private boolean isTerminalD4(IAtom atom) {
```

Comment:
Just a side note, an `-S(=O)(=O)-` as a perpendicular cross (four 90 degree angels )would be nice too (also for P).

Just a future feature request :)

Reply:
Yes but as it's in a chain it's better to kink it IMO:

![image](https://user-images.githubusercontent.com/983232/62155401-ad471700-b300-11e9-9aa5-48a9e260a341.png)
![image](https://user-images.githubusercontent.com/983232/62155446-d36cb700-b300-11e9-8309-75dae10a9be9.png)

Reply:
Subjective though

Reply:
Maybe an Option? (I think the above is ecstatically wrong :)

Reply:
Alex Clark has a nice way of handling this with functional group patterns, this also allows you to depict see-saw like arrangements for example. That would be a big change though to the atom placement.